### ISSUE TITLE
Expose a `saveThemesToFile` function

### ISSUE THREAD

**Issue by corysimmons** [vscodethemes/web#269]
Hi, would you be interested in exposing a function that saves all the color palettes to a file with this structure:

```json
[
  { name: "GitHub Theme", colors: { background: "#000", foreground: "#fff", etc... } },
  { name: "PowerShell", colors: { ... } }
]
```

I think it'd be nice if a bunch of the most popular VSCode Themes were neatly organized in a single JSON file.

My specific use case is there is a site at https://ui.jln.dev ([GitHub](https://github.com/jln13x/ui.jln.dev)) where you can generate palettes for shadcn-ui (a popular Next.js UI component library), and I think it'd be neat to see what those components look like with VSCode Themes on them.

I tried to get these myself ([discussion](https://github.com/jln13x/ui.jln.dev/discussions/9)), but it gets pretty complicated with how theme developers generate their palettes. It looks like you've already handled all those cases so it'd be great if we could just use your work.

Thank you.

**Comment by jschr** [comment]
hey, cool project! 

If I provided you a data export would that help in the short term? Longer term I could consider a exposing an API for this.

**Comment by corysimmons** [comment]
Yeah actually a data export is realistically probably all we'll need. Thank you!!

**Comment by jschr** [comment]
I contacted you through your website to get your email. I'll follow up with a data export

**Comment by corysimmons** [comment]
Thank you Jordan!
